x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/utils/custom_sort_script.ts

```diff
@@ -28,7 +28,7 @@ export const severitySortScript = (direction: string) => ({
     script: {
       lang: 'painless',
       inline:
-        "if(params.scores.containsKey(doc['vulnerability.severity'].value)) { return params.scores[doc['vulnerability.severity'].value];} return 1000;",
```

Comment:
missing null check was causing the search to crash when severity is not present on a document.

Reply:
@Omolola-Akinleye as far as I can see you worked on this script. was there a specific requirement to have `1000` as the sorting value for unknown severity? Just checking that I'm not missing any use case by changing it to 0 as per this comment https://github.com/elastic/security-team/issues/7146#issuecomment-1669839676